site/src/pages/content/button-guidelines.mdx

```diff
@@ -5,6 +5,11 @@ description: Use this guide to make your labels predictable and consequential. Y
 
 Use this guide to make your labels predictable and consequential. Your users depend on them, give it some thought and time.
 
+import Rules from '../../../../gatsby-theme-mds/src/components/Rules/Rules.js';
+import DOs from '../../../../gatsby-theme-mds/src/components/Rules/DOs.js';
+import DONTs from '../../../../gatsby-theme-mds/src/components/Rules/DONTs.js';
+import InlineMsg from '../../../../gatsby-theme-mds/src/components/Rules/InlineMsg.js';
```

Comment:
User should not need to import the components in mdx file, it should be available through the mdx provider , you can refer https://github.com/innovaccer/mds-docs/blob/master/gatsby-theme-mds/src/components/Layout.js#L27